bpf/bpf_lxc.c

```diff
@@ -790,6 +802,14 @@ ipv6_policy(struct __sk_buff *skb, int ifindex, __u32 src_label, __u8 *reason, s
 
 	*reason = ret;
 
+	// Check it this is return traffic to an egress proxy.
+	// Do not redirect again if the packet is coming from the egress proxy.
+	if ((ret == CT_REPLY || ret == CT_RELATED) && ct_state.proxy_redirect &&
+	    !tc_index_skip_egress_proxy(skb)) {
```

Comment:
I'm confused. This function is `ipv6_policy()` which is called from `tail_ipv6_policy()` and `tail_ipv6_to_endpoint()`, ie the endpoint ingress case. Why is an egress proxy skip bit being checked here in the ingress path?

Reply:
See above.